### ISSUE TITLE
make build not working

### ISSUE THREAD

**Issue by JavierGOrdonnez** [ITISFoundation/jupyter-neuron#2]
Directly cloning and, without any modification, running `make build` leads to the following error:
![image](https://user-images.githubusercontent.com/56032114/231479871-8b29c353-4031-4535-ad11-0eefcfa1f46c.png)

I have checked and the installed version of octave (with `apt-get install octave`) is 5.2.0-1.
My guess is that the `statistics` package has been updated but the core `octave` has not, thus leading to an incompatibility.

I will try to find a solution to install octave >= 6.1.0 (for my own service), and post it here.

**Comment by JavierGOrdonnez** [comment]
apt list -a octave gives only version 5.2 - it does not seem possible to install version 6.1, 7 or 8
Thus, solution seems to be to comment out the line installing statistics:
![image](https://user-images.githubusercontent.com/56032114/231684722-8e6c8e90-d3f5-4e8a-8e91-161200e049cc.png)

**Comment by JavierGOrdonnez** [comment]
Package incompatibility arising in `requirements.in`:
![image](https://user-images.githubusercontent.com/56032114/231685020-fbc2b87a-338c-4253-8920-d07bfaacae7a.png)

Content of requirements.in:
# math-base
h5py
matplotlib
numpy
plotly
scikit-learn
scipy
sympy

pandas
# optional extensions for pandas
openpyxl
xlrd
xlwt

# jupyter-extensions
ipyevents
ipympl
ipywidgets
jupyterlab_latex
jupyterlab_widgets
jupyterlab-git
jupyterlab-lsp
octave_kernel
python-lsp-server[all]
voila
watchdog[watchmedo]

# NEURON Simulator & co.
neuron
pynn
brian2
netpyne
brian2tools

**Comment by JavierGOrdonnez** [comment]
Specifying this package version solved it: `ypy-websocket==0.8.2`.

To ensure full compatibility, I would propose to do:
```
ypy-websocket==0.8.2
jupyter-ydoc==0.2.4
jupyterlab-lsp==4.0.1
```

**Comment by JavierGOrdonnez** [comment]
Only Octave kernel seems available: 
![image](https://user-images.githubusercontent.com/56032114/232765771-a49d3b1b-9e20-430d-b119-f602d6d4b0d9.png)

Also, permisions are not correctly set, and no new notebook can be created in `workspace`: 
![image](https://user-images.githubusercontent.com/56032114/232765963-4784fdf0-ef58-45e4-ade1-c3f5cb078f7a.png)
